testcontainers/src/main/java/org/jdbi/v3/testing/junit5/tc/TestcontainersDatabaseInformationSupplier.java

```diff
@@ -41,6 +42,7 @@ final class TestcontainersDatabaseInformationSupplier implements Supplier<Testco
     private final SynchronousQueue<TestcontainersDatabaseInformation> nextSchema = new SynchronousQueue<>();
     private final AtomicBoolean closed = new AtomicBoolean();
     private final CountDownLatch stopped = new CountDownLatch(1);
+    private final ReentrantLock mutex = new ReentrantLock();
```

Comment:
you don't need the mutex. The CountDownLatch right next door does the exact same thing...